intellij-plugin-verifier/verifier-test/mock-plugin/src/main/resources/mock/plugin/property/linter.properties

```
@@ -0,0 +1,2 @@
+mock.plugin.linter.quiet=Quiet Linter
+mock.plugin.linter.verbose=verbose Linter
```

```suggestion
mock.plugin.linter.verbose=Verbose Linter
```
I know it is insignificant, but still 🙃